ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -260,7 +323,14 @@ func verifyConvertToContainerConfigOutput(t *testing.T, expected types.ServiceCo
 	} else {
 		assert.Empty(t, actual.MountPoints)
 	}
-	// TODO: verify expected.Environment
+
+	if len(expected.Environment) > 0 {
+		for _, env := range actual.Environment {
+			assert.Equal(t, *expected.Environment[*env.Name], *env.Value)
```

Comment:
Nit: If you wanted, you can use `aws.StringValue` here (does the opposite of `aws.String`), but this works as well.